src/bundle/Resources/public/js/alloyeditor/src/plugins/ez-custom-tag.js

```diff
@@ -210,6 +210,8 @@
                  */
                 setName: function(name) {
                     this.element.data('ezname', name);
+                    window.clearTimeout(this.setNameFireEditorInteractionTimeout);
```

Comment:
Each timeout handler should be defined at the top (after the `editables` object).
```js
setNameFireEditorInteractionTimeout: null
```